torch/lib/THD/base/data_channels/DataChannelTCP.cpp

```diff
@@ -70,30 +69,26 @@ DataChannelTCP::DataChannelTCP(InitMethod::Config config, int timeout)
   , _port(0)
   , _timeout(timeout)
   , _poll_events(nullptr)
+  , _processes(config.world_size)
 {
   _rank = config.rank;
 
-  if (_rank == MASTER_RANK) { // MASTER
+  if (_rank == 0) { // MASTER
```

Comment:
1. Being consistent is not a bad idea
2. I do not think someone is going to change it somewhere in the future because it makes more harm than good. (0 is very descriptive in my opinion and clear - MPI use it, so we should too)
3. Believe me, those 3-5 places where this variable is used, is nothing comparing to ~50 (or even more) where we use just 0.